File: 251229/코드네임/code-name.py
class Agent:
    def __init__(self, code_name, score):
        self.code_name = code_name
        self.score = score

arr = []
for _ in range(5):
    code_name, score = input().split()
    score = int(score)
    arr.append(Agent(code_name, score))

min_val = arr[0]
for agent in arr:
    if agent.score <= min_val.score:
        min_val = agent


print(min_val.code_name, min_val.score)

File: 251229/코드네임/code-name.js
class Agent {
    constructor(codeName = 0, score = 0) {
        this.codeName = codeName;
        this.score = score;
    }
}

const fs = require('fs');
const input = fs.readFileSync(0).toString().trim().split('\n');

const agentList = [];
for (let i = 0; i < 5; i++) {
    const [codeName, score] = input[i].split(' ');
    agentList.push(new Agent(codeName, Number(score)));
}

let min = agentList[0];
for (let i = 0; i < 5; i++) {
    if (min.score >= agentList[i].score) {
        min = agentList[i];
    }
}

console.log(min.codeName, min.score);